doc: Don't checkout to sdk-nrf main (#447)
nrfconnect/sdk-sidewalk: doc/setting_up_sidewalk_environment/setting_up_sdk.rst

       |___ zephyr
       |___ ...

> #. Check manifest file. This step is necessary if you worked with Sidewalk as application with manifest repository before.

Reviewer: ```suggestion
#. If you have worked with the Sidewalk application in a previous distribution model (as a manifest repository), check the manifest file.
   Make sure the manifest path points to :file:`west.yml` inside the :file:`nrf` directory:
```
Not sure if the meaning is conveyed - please let me know if something doesn't seem right

>    Check the current manifest path and make sure it points to :file:`west.yml` inside the :file:`nrf` directory:

Reviewer: ```suggestion
```

Can we rephrase the current line 73 - ``If it does not, set the manifest path by running the following command:`` ? Changed the main sentence a bit so the following needs to match it gramatically, plus felt it could use some rewording.

       |___ zephyr
       |___ ...

#. If you have worked with the Sidewalk application in a previous distribution model (as a manifest repository), check the manifest file.
   Make sure the manifest path points to :file:`west.yml` inside the :file:`nrf` directory:

      .. code-block:: console

> 

Reviewer: Would it be possible to apply through this PR the second comment I left (line 73)?

Author: Rephrased the sentence, I hope now it fits better

       |___ zephyr
       |___ ...

#. If you have worked with the Sidewalk application in a previous distribution model (as a manifest repository), check the manifest file.

   Make sure the manifest path points to :file:`west.yml` inside the :file:`nrf` directory:

      .. code-block:: console

         $ west manifest --path
         /path-to-ncs-folder/nrf/west.yml

>    In case your manifest path points to different file, use the following command:

Reviewer: ```suggestion
   In case your manifest path points to a different file, use the following command:
```
